devel/playdate-sdk/Portfile

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    playdate-sdk
+version                 1.12.3
+revision                0
+categories              devel
+maintainers             {idleberg @idleberg} openmaintainer
+license                 Restrictive
+
+description             Playdate SDK
+long_description        This package installs includes a rich set of APIs, a \
+                        compiler    , and documentation for Playdate game development.
+
+homepage                https://play.date/dev/
+master_sites            https://download-keycdn.panic.com/playdate_sdk/
+distname                PlaydateSDK-${version}
+distfiles               ${distname}.zip
+use_zip                 yes
```

Comment:
Minor nit-pick: Move `use_zip yes` above the line for `distfiles`, then change the latter to:

```
distfiles         ${distname}${extract.suffix}
```

That's the preferred way to go, rather than hard-coding the file extension.

Reply:
I don't think setting "distfiles" is needed at all - right?

Reply:
> I don't think setting "distfiles" is needed at all - right? Isn't what you show the default, only setting "distname" should be enough I think.

Yes, you're right. As usual, I'm compartmentalizing a bit too much. Lol